Models/CustomCard.cs

```diff
@@ -58,20 +57,6 @@ public CustomCard(string name, List<AbilityIdentifier> abilityId=null, EvolveIde
 
 			// Handle AbilityIdentifier
 			List<AbilityIdentifier> toRemove = new List<AbilityIdentifier>();
-			if (this.abilityId is not null)
```

Comment:
This code block is not needed since IdentifierHandlers will add the ability to the cards.

Reply:
Handling this on card addition could be better in some very niche cases and won't have any effect on load time so may as well do it here if possible too

Reply:
Added back in.